google-cloud-bigquery/acceptance/bigquery/dataset_access_test.rb

```diff
@@ -161,4 +162,292 @@
       refute dataset.access.reader_routine? routine
     end
   end
+
+  describe :condition do
+    let(:dataset_id) { "#{SecureRandom.hex(4)}_dataset" }
+    let(:dataset) do
+      d = bigquery.dataset dataset_id, access_policy_version: 3
+      if d.nil?
+        d = bigquery.create_dataset dataset_id, access_policy_version: 3
+      end
+      d
+    end
+
+    let(:condition) do
+      Google::Cloud::Bigquery::Condition.new(
+        "true",
+        description: "example description",
+        location: "path/to/example/location",
+        title: "example title"
+      )
+    end
+    let(:c) { condition.to_gapi.to_h }
```

Comment:
Let's not use names like this. Ruby, unlike Go, prefers more descriptive names. For example, `condition_as_hash`.